lib/platform-stats.js

```diff
@@ -0,0 +1,26 @@
+import createDebug from 'debug'
+
+const debug = createDebug('spark:platform-stats')
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('./preprocess').Measurement[]} honestMeasurements
+ */
+export const updatePlatformStats = async (pgClient, honestMeasurements) => {
+  await updateDailyNodeMetrics(pgClient, honestMeasurements)
+}
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('./preprocess').Measurement[]} honestMeasurements
+ */
+export const updateDailyNodeMetrics = async (pgClient, honestMeasurements) => {
+  debug('Updating daily node metrics, count=%s', honestMeasurements.length)
+  for (const m of honestMeasurements) {
```

Comment:
Logging the number of measurements is not very useful. We should log the number of unique Station ids found in the measurements.

It's also important to extract this list of unique IDs before we run SQL queries. Your original implementation will run one SQL query for each measurement, that will perform very poorly.

I would like you to make the following changes:

1. First, build a list of unique station IDs. See how we collect unique participant addresses. https://github.com/filecoin-station/spark-evaluate/blob/562125a2c1bc99b4aaf0fceba4c4bb0a872913b5/lib/public-stats.js#L16-L25
2. Log the number of unique Station IDs found.
3. Use the list of unique Station IDs as the input to SQL queries.

Reply:
Committed! I assumed we don't need the extra logic around collecting unique particpant addresses, since the IDs are already provided to use rather than a serial SQL field. Let me know if this works for you!

Reply:
I merged in the latest `main` branch that includes https://github.com/filecoin-station/spark-evaluate/pull/196, we will get the dry-run results soon from the CI.

Reply:
Oh, we need also need to fix our GHA triggers, see https://github.com/filecoin-station/spark-evaluate/pull/198